src/prediction.jl

```diff
@@ -149,44 +177,321 @@ function unstack_techreplicates(raw_data, retain_first=true)
     return unstackedDf
 end #end function
 
-function report_classification_merit(classification_results::Dict)
-    classification_merits = DataFrame(
-    :Trial => collect(1:classification_results[:n_trials]),
-    :Train_ACC => classification_results[:train_accuracies],
-    :Test_ACC => classification_results[:test_accuracies],
+####################################
+# 2. Training/Processing Functions #
+####################################
+
+RandomForestRegressor = MLJ.@load RandomForestRegressor pkg=DecisionTree
+RandomForestClassifier = MLJ.@load RandomForestClassifier pkg=DecisionTree
+
+function train_randomforest(
+    type::Regression,
+    ref_name::String,
+    original_df,
+    input_cols,
+    output_col;
+    n_splits = 2,
+    data_preparation_fun = identity,
+    tuning_space = (
```

Comment:
```suggestion
    tuning_space = (;
```

Just making NamedTuple explicit

Reply:
I didn't know that the `NamedTuple` constructor worked like that (names are actually keyword arguments). I will alter the occurrences of the tuning_space accordingly. Thank you.